oat-sa/tao-calculator-fe
# Feature/TR-5071/Add instant computation mode

### Related to: https://oat-sa.atlassian.net/browse/TR-5071

### Summary

Requires: https://github.com/oat-sa/tao-calculator-fe/pull/19

PR 5/5 for adding the instant computation mode to the calculator:
- Add the API for activating the instant computation mode

Please see the details below for more info.
Other PR will come later for completing the feature.

### Details

Several changes have been made:
- add an API for setting the instant computation mode
- limit and replace strategies have been fixed/improved
- pipe the strategies for computing the result when needed
- the sandbox has been improved for allowing to set the instant mode and for ease the debugging

### How to test
- run the unit tests: `npm test`
- open the sandbox and play with the instant mode: `npm run dev`
    - activate it using the switch
    - see how the expression is computed as you enter terms
